stdlib/3/itertools.pyi

```diff
@@ -19,7 +20,17 @@ def repeat(object: _T) -> Iterator[_T]: ...
 @overload
 def repeat(object: _T, times: int) -> Iterator[_T]: ...
 
-def accumulate(iterable: Iterable[_T], func: Callable[[_T, _T], _T] = ...) -> Iterator[_T]: ...
+if sys.version_info >= (3, 8):
+    @overload
+    def accumulate(iterable: Iterable[_T],
+                   func: Callable[[_T, _T], _T] = ...) -> Iterator[_T]: ...
+    @overload
+    def accumulate(iterable: Iterable[_T],
+                   func: Callable[[_S, _T], _S],
+                   initial: Optional[_S] = ...) -> Iterator[_S]: ...
```

Comment:
I think there should be no default for ` = ...` because otherwise the overloads will overlap.

Reply:
Thanks for your review. I see your point.